src/TableRow.jsx

```diff
@@ -0,0 +1,44 @@
+import moment from "moment";
+import React, { useState } from "react";
+import CustomerProfile from "./CustomerProfile";
+
+
+const TableRow=(props)=>{
+    const [selectedRow, setSelectedRow] = useState(null);
+    const [showProfile, setShowProfile] = useState(false);
+    const [customClass, setCustomClass] = useState("");
+
+    let checkIn = moment(props.value.checkInDate, "YYYY-MM-DD")
+    let checkOut = moment(props.value.checkOutDate , "YYYY-MM-DD")
+    let rowClass = selectedRow === props.value.id ? customClass : "";          
+
+    const toggleProfile = () => {
+    setShowProfile((showProfile) => !showProfile);
+  };
+
+  function classHandler(rowId) {
+    setSelectedRow(rowId);
+    setCustomClass((oldCustomClass) => (oldCustomClass ? "" : "table-primary"));
+  }
```

Comment:
We have a mix of functions, and arrow functions in this component. Both are fine, but we should be consistent and pick one type.